tests/keyboard-test/UbuntuKeyboard/Dummy.qml

```diff
@@ -1,5 +0,0 @@
-import QtQuick 2.0
-
-Item {
-    property bool available: true
-}
```

Comment:
We should only remove that if we remove the corresponding C++ code, I think. Same for qmldir just below.